interface/batchcom/emailnotification.php

```diff
@@ -36,8 +61,7 @@
     if ($_POST['provider_name']=="") $form_err.=xl('Empty value in "Name of Provider"','','<br>');
     if ($_POST['message']=="") $form_err.=xl('Empty value in "Email Text"','','<br>');
     //process sql
```

Comment:
deal with sql injection:
``````
//process sql
    if (!$form_err) {
    $next_app_time = $_POST[hour].":".$_POST['min'];
    $sql_text=" ( `notification_id` , `sms_gateway_type` , `next_app_date` , `next_app_time` , `provider_name` , `message` , `email_sender` , `email_subject` , `type` ) ";
    $sql_value=" ( ?, ?, ?, ?, ?, ?, ?, ?, ? ) ";
    $query = "REPLACE INTO `automatic_notification` $sql_text VALUES $sql_value";
    //echo $query;
    $id = sqlInsert($query, array($_POST[notification_id], $_POST[sms_gateway_type], $_POST[next_app_date], $next_app_time, $_POST[provider_name], $_POST[message], $_POST[email_sender], $_POST[email_subject], $type) );
    $sql_msg="ERROR!... in Update";
    if($id)    $sql_msg="Email Notification Settings Updated Successfully";
   }